### DISCUSSION TITLE
Getting Warning: Each child in a list should have a unique "key" prop.

### DISCUSSION THREAD

**Discussion by wwsskkaa** [TanStack/table#2716] [General]
I keep getting this warning 
console.error node_modules/react/cjs/react.development.js:175
      Warning: Each child in a list should have a unique "key" prop.

but I thought those getHeaderProps(), getRowProps()... etc actually return a key prop? 
anyone knows how to stop getting this warning?

**Comment by moneebalalfi** [comment]
Anyone has solved this problem ?

**Comment by MaybeRex** [comment]
just wrap your child elements in a div and give that a key
